_includes/header.html

```diff
@@ -30,7 +30,7 @@
         <ul>
           <li class="link"><a href="{{ site.baseurl }}/countries">Countries</a></li>
           <li class="link"><a href="{{ site.baseurl }}/indicators">Data</a></li>
-          <li class="link"><a href="{{ site.baseurl }}/countries#compare">Compare</a></li>
+          <li class="link"><a href="{{ site.baseurl }}/compare">Compare</a></li>
```

Comment:
Can you take a look of where we are linking to this section and change the link?
Places which come to my mind: footer, country section...